### PR TITLE
fix: #330 extra quotes on string test

### PR DESCRIPTION
**Pull request by zekth**
fix: #330 

Extra quotes were added when using the test serializer leading to the null fallback

### CODE REVIEW

**Review by jsumners — APPROVED** [review]
Looks good to me.

File: test/anyof.test.js [hunk 1]
```diff
@@ -471,3 +471,26 @@ test('anyOf object with field of type string with format or null', (t) => {
     prop: toStringify
   }), `{"prop":"${toStringify.toISOString()}"}`)
 })
+
+test('anyOf object with field date of type string with format or null', (t) => {
+  t.plan(1)
+  const toStringify = '2011-01-01'
```
Comment (Eomm) [inline test/anyof.test.js:477]: I would add a test with a `foo bar` string and add a disclaimer in the readme that a date format string, will not be validated (because we assume that the user has already parsed the Date object)

**Review by Eomm — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Review by L2jLiga — APPROVED** [review]
LGTM
